htdocs/societe/list.php

```diff
@@ -1091,7 +1091,8 @@
 	}
 	if (! empty($arrayfields['s.phone']['checked']))
 	{
-		print "<td>".$obj->phone."</td>\n";
+		$tmpcode=getCountry($obj->fk_pays,2);
```

Comment:
This create a useless SQL request at each pass in the loop.
You can get $tmpcode by adding the field "code" of table country inside the previous select.